transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/http/RestClientInvocation.java

```diff
@@ -274,7 +274,7 @@ protected void fail(ConnectionBase connection, Throwable e) {
 
   protected void setCseContext() {
     try {
-      String cseContext = JsonUtils.writeValueAsString(invocation.getContext());
+      String cseContext = JsonUtils.writeUnicodeValueAsString(invocation.getContext());
```

Comment:
Is this compatible with old versions? e.g. new consumer invoke an older version provider.

Reply:
I think it is compatible, Jackson will automatic decode unicode string in deserialization.
In testcase, use writeUnicodeValueAsString to serialize and use the old OBJ_MAPPER to deserialize, the value is not changed

Reply:
Can you add some unit test case for this?

Reply:
ok, i added 2 testcase in TestRestClientInvocation.java